histomicstk/ComputeHaralickFeatures.py

```diff
@@ -0,0 +1,199 @@
+import numpy as np
+import pandas as pd
+import skimage.feature
+from skimage.measure import regionprops
+
+
+def ComputeHaralickFeatures(I, Label, Offsets=[[0, 1]], NumLevels=256,
+                            MaxGray=255, MinGray=0):
```

Comment:
Yes. binning strategies can be very useful for many tasks, such as feature extraction and segmentation. I think we need to talk a little about what is the optimal number of gray levels for the project tomorrow.